Dockerfile

```diff
@@ -0,0 +1,39 @@
+ARG ROS_DISTRO=iron
+
+FROM ros:${ROS_DISTRO}
+
+# This is pushed to the docker hub to be accessible to the GitHub Actions
+
+# Install the golang package
+RUN apt update && apt install -y \
+  wget
+
+# Install the specific golang version
+RUN cd /tmp \
```

Comment:
The way we install Go in other docker files is to copy the FS from the image:

```
COPY --from=golang:1.23.0-bookworm --chmod=777 /usr/local/go/ /usr/local/go/
```

I don't have a strong preference for this method, but it might be better to use the tried and true one. If you'd like to stick to downloading and un-taring it, I'd verify that file permissions are indeed 777 once you untar

Reply:
It didn't click until 30 seconds ago that this was copying from the `golang:1.23.0-bookworm` Docker image.

I was so confused that this was working when I didn't have Golang installed on my host machine.

Happy to make that change